ginga/examples/gw/shared_canvas.py

```diff
@@ -308,31 +307,24 @@ def main(options, args):
 
 if __name__ == "__main__":
 
-    # Parse command line options with nifty optparse module
-    from optparse import OptionParser
-
-    usage = "usage: %prog [options] cmd [args]"
-    optprs = OptionParser(usage=usage, version=('%%prog'))
-
-    optprs.add_option("--debug", dest="debug", default=False,
-                      action="store_true",
-                      help="Enter the pdb debugger on main()")
-    optprs.add_option("--log", dest="logfile", metavar="FILE",
```

Comment:
`--log`, `--loglevel`, and `--stderr` removed on purpose?

Reply:
yes, because I added `log.addlogopts(argprs)` call below to add those options.